tests/test_services.py

```
@@ -83,17 +85,72 @@ def test_optimization_service_handles_solver_failure():
+            config.COL_NAME: ["P1", "P1"],
+            "Score1": [10, 10],
+            config.COL_GROUPER: ["", ""],
+            config.COL_SEPARATOR: ["", ""],
+        }
+    )
+    weights = {"Score1": 1.0}
+
+    res1, _ = OptimizationService.run(
+        data, [1, 1], weights, ConflictPriority.GROUPERS, 10
+    )
+    res2, metrics2 = OptimizationService.run(
```

Ruff (0.15.12)

[warning] Unpacked variable `res2` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)